### DISCUSSION TITLE
Polybar not launching with launch.sh

### DISCUSSION THREAD

**Discussion by jnptk** [polybar/polybar#3063] [Q&A]
I just got into customizing wm's and installed polybar. When I do `polybar` command in the terminal it launches, but not with the `launch.sh` that I created under _$HOME/.config/polybar/launch.sh_. I configured polybar to launch on both my monitors so its pretty easy to see if its working or not. Launching polybar by running the file directly works fine. I couldn't find anything about declaring the launch file some where so, anybody got an idea what I am doing wrong?

Thanks in advance.

**Comment by patrick96** [comment]
I'm not sure I understand. Does running `launch.sh` work?

If not, there should be some error messages somewhere (depending on how the `launch.sh` looks like).

It it does run, what exactly is the problem?
